src/unix/hermit/mod.rs

```diff
@@ -19,7 +19,9 @@
 pub type c_long = i64;
 pub type c_ulong = u64;
 
+#[allow(unused)]
```

Comment:
So this is weird, the only way in which these could be unused is if they are not exported by the library API, right?